client/src/test/java/org/spine3/base/CommandsShould.java

```diff
@@ -64,6 +69,11 @@
     private final StringValue stringValue = newStringValue(newUuid());
 
     @Test
+    public void have_private_ctor() {
```

Comment:
Could you please add one more test using the `NullToleranceTest` utility to ensure the `Commands` methods aren't vulnerable?